### ISSUE TITLE
v14 date timezone offset incorrect

### ISSUE THREAD

**Issue by shearer3000** [umbraco/Umbraco-CMS#17001]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

14.2.0

### Bug summary

The values displayed for the node "Created" date and in the History pane dates are incorrect (appear to be incorrectly applying UTC offset).

For example, given New Zealand time (NZST) which is currently UTC +12 - its 10:45am and the history shows published at 10:45PM:

![image](https://github.com/user-attachments/assets/5aa7fc56-7458-4641-8bd2-939d79cb84bb)

### Steps to reproduce

1. Create node - Created date shows "correct" time:
![image](https://github.com/user-attachments/assets/767f7194-a09f-48df-a741-c583c6a32e0d)

2. Save and publish

3. Observe Created date and History pane now showing incorrect date (12 hrs in the future of current timezone):

![image](https://github.com/user-attachments/assets/36d0d7b9-a020-4eb7-b81f-b796f20ffb24)
![image](https://github.com/user-attachments/assets/81d47a58-2b21-4a04-94e2-d2880cdf2e74)

### Expected result / actual result

Expected: dates reflect current timezone
Actual: dates appear to show timezone offset incorrectly

---
_This item has been added to our backlog AB#47442_

**Comment by NguyenThuyLan** [comment]
Hi @shearer3000 , thanks for reporting. Unfortunately, I couldn't reproduce this issue like this

https://github.com/user-attachments/assets/15f3aad9-c463-443e-aa40-138cbd36ebb3

**Comment by shearer3000** [comment]
Hi @NguyenThuyLan thanks for investigating and providing that video. What timezone are you (i.e. your offset to UTC)? I just tried again on a fresh install on 14.2.0 and most definitely see this issue consistently. Im in New Zealand (so a 12h UTC offset) - its currently 9:50am and publishing a page shows a PM time. Please also see my system clock in the screenshot to contrast with the "future" date Umbraco is displaying:

![image](https://github.com/user-attachments/assets/63ab9af7-090e-4759-8748-62be378353b3)

**Comment by NguyenThuyLan** [comment]
@shearer3000 , you're right, last time, I tried on Cloud and it ran correctly, but when creating locally, there was a timezone error here.

**Comment by NguyenThuyLan** [comment]
Hi @shearer3000 👋, After testing it again with the latest versions (v14.3.1 - v15.1.1) I can no longer reproduce the problem. 🤷‍♀️
Given this, I’ll proceed with closing this issue. If you encounter it again or experience any related problems, please feel free to reopen this issue.
